SoObjects/SOGo/SOGoSource.h

```diff
@@ -118,4 +118,10 @@
 - (void) updateBaseDNFromLogin: (NSString *) theLogin;
 @end
 
+@protocol MembershipAwareSource <SOGoSource>
```

Comment:
To be consistent with the other protocols, add the SOGo prefix (`SOGoMembershipAwareSource` or simply `SOGoMembershipSource`).

Reply:
Resolved in commit 948c89f